<?xml version="1.0"?>
<launch>
    <arg name="robot" default="test"/>
    <arg name="reach_object" default="test"/>
    <arg name="config_name" default="test"/>

    <param name="visualize_results" value="false"/>
    <include file="$(find reach_core)/launch/$(arg robot).launch"/>

    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>

    <node name="robot_reach_study_node" pkg="reach_core" type="robot_reach_study_node" output="screen" required="true">
      <rosparam command="load" file="$(find reach_core)/input/$(arg reach_object).yaml"/>
      <param name="config_name" value="$(arg config_name)"/>
    </node>
    <node name="sample_mesh_server" pkg="reach_core" type="sample_mesh_server" output="screen"/>

</launch>
